docs/index.rst

```diff
@@ -1,27 +1,24 @@
-=======
-Shapely
-=======
+.. include:: ../README.rst
+
+.. include:: ../CREDITS.txt
```

Comment:
This is long enough that it probably would do better on its own top level page.  Alternatively, we could instead adopt the approach of numpy and pandas and list contributors for each release, but not maintain a single long list (per-release seems easier from a maintenance perspective).  This list could then go into a dedicated "contributors prior to 2.0" page (not quite sure where).